views/layouts/default.twig

```diff
@@ -59,7 +66,7 @@
                 <li><a href="{{ path('conversation.index') }}">Inbox{% if unread > 0 %} ({{ unread }}){% endif %}</a></li>
                 <li><a href="{{ path('auth.logout') }}">Logout</a></li>
             {% else %}
-                <li><a href="{{ path('auth.login') }}">Login</a></li>
+                <!-- <li><a href="{{ path('auth.login') }}">Login</a></li> -->
```

Comment:
This is currently commented out in production, so I commented it out in this PR.

Reply:
I noticed. 😉